### PR TITLE
Adds hearable/visible emotes again

### PR DESCRIPTION
**Pull request by Archanial**
## Changelog
:cl:
add: You can now choose whether your emote is visible or audible whem using "me speak".
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
Does this dialog open every fucking time you press the M key? why not just make a new verb for audible or visible, because this just seems really fucking annoying

File: code/modules/mob/say.dm [hunk 1]
```diff
@@ -34,7 +34,15 @@
 
 	message = trim(copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN))
 
-	usr.emote("me",1,message,TRUE)
+	var/m_type = alert(usr, "Choose the emote type", "Emote type", "Audible emote", "Visible emote")
```
Comment (Archanial) [inline code/modules/mob/say.dm:40]:
```suggestion
	if(!message)
		return

	var/m_type = alert(usr, "Choose the emote type", "Emote type", "Audible emote", "Visible emote")
```

**Comment by Archanial** [comment]
> 
> 
> Does this dialog open every fucking time you press the M key? why not just make a new verb for audible or visible, because this just seems really fucking annoying

well yes

**Comment by Kapu1178** [comment]
> 
> 
> > Does this dialog open every fucking time you press the M key? why not just make a new verb for audible or visible, because this just seems really fucking annoying
> 
> well yes

Thats.... awful.

Make them separate verbs?

**Pull request closed without merging**